tests/unit/test_mxnet.py

```diff
@@ -175,7 +175,7 @@ def _create_compilation_job(input_shape, output_location):
 
```

Comment:
Are there similar tests for pytorch?

Reply:
No, we only have unit tests for pytorch. It's debatable if we have a gap there since the mxnet test is already testing the part of the logic that calls the neo service to compile model. We are planning to revamp our build system to handle more parallel integ testing. Once the system is more scalable we will be in a better position to add integ tests.